src/printer.rs

```diff
@@ -29,34 +29,47 @@ struct SerializedNpmPkg<'a> {
   dependencies: Vec<Cow<'a, str>>,
 }
 
+// WARNING: It's important to implement Ord/PartialOrd on the final
+// normalized string so that sorting works according to the final
+// output and so that's why this is used rather than JsrDepPackageReq
+// directly.
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Serialize)]
```

Comment:
Great, there's a warning, is there a test that asserts that as well?

Reply:
I'll try to come up with one.

Reply:
Can't think of one atm 😅